statsmodels/robust/scale.py

```diff
@@ -49,6 +49,46 @@ def mad(a, c=Gaussian.ppf(3/4.), axis=0, center=np.median):
     return np.median((np.abs(a-center)) / c, axis=axis)
 
 
+def iqr(a, c=Gaussian.ppf(3/4) - Gaussian.ppf(1/4), axis=0, center=np.median):
+    # c \approx 1.349
+    """
+    The Interquartile range along given axis of an array
+
+    Parameters
+    ----------
+    a : array_like
+        Input array.
+    c : float, optional
+        The normalization constant, used to get consistent estimates of the
+        standard deviation at the normal distribution.  Defined as
+        scipy.stats.norm.ppf(3/4.) - scipy.stats.norm.ppf(1/4.), which is
+        approximately 1.349.
+    axis : int, optional
+        The default is 0. Can also be None.
+    center : callable or float
+        If a callable is provided, such as the default `np.median` then it
+        is expected to be called center(a). The axis argument will be applied
+        via np.apply_over_axes. Otherwise, provide a float.
+
+    Returns
+    -------
+    iqr : float
+        `mad` = median(abs(`a` - center))/`c`
```

Comment:
Looks more like a note then a return docstring. Can just same The normalized  interquartile range

Reply:
Fixed in commit ce3770f2774fe3c61fd7b1e5c1d4657d3731e1bf